### DISCUSSION TITLE
Ha Integration wird nicht gefunden

### DISCUSSION THREAD

**Discussion by Shwowak** [marq24/ha-evcc#46] [Q&A] [answered]
Ich habe versucht die Integration Über HACS zu installieren es wird leider nicht gefunden kann mir jemand helfen ?

**Comment by marq24** [comment]
... damit Dir jemand helfen könnte müsstest Du schon beschreiben, WAS Du Wo versucht hast... und WAS dann genau für Dich nicht funktioniert hat...

**Reply by Shwowak** [reply]
Ich habe noch nicht gemacht, da ich es nicht finden kann.

**Comment by Shwowak** [comment]
<img width="1280" alt="Bildschirmfoto 2024-11-15 um 07 08 42" src="https://github.com/user-attachments/assets/60f9c8ab-3774-4ad3-8526-30e2ce367641">

**Reply by UDicke** [reply]
gib doch bei der Suche einfach mal nur "evcc" oder "marq24"

**Comment by Shwowak** [comment]
![2](https://github.com/user-attachments/assets/67ee40c7-796c-40ca-b0fb-1495ad785be6)
![1](https://github.com/user-attachments/assets/1b0dbf19-bdde-4fd0-ba7c-5ccab78d0a00)
Das habe ich schon gemacht.

**Comment by marq24** [comment, marked as answer]
Schritt 1) Füge das CUSTOM REPOSITORY zu HACs hinzu... -> das geht über die 3 Punkte oben in der Ecke...

**Comment by Shwowak** [comment]
Super Danke
